packages/web/app/views/programRegistry/PatientProgramRegistryView.jsx

```diff
@@ -107,7 +108,10 @@ export const PatientProgramRegistryView = () => {
           <DisplayPatientRegDetails patientProgramRegistration={data} />
           <Grid>
             <ProgramRegistryStatusHistory />
-            <ConditionSection registrationId={data?.id} />
+            <ConditionSection
+              registrationId={data?.id}
+              isInactive={data?.registrationStatus === REGISTRATION_STATUSES.INACTIVE}
```

Comment:
Would this apply to `RECORDED_IN_ERROR` too?

Reply:
no since they are not viewable on the font end